R/matching.R

```diff
@@ -38,7 +45,7 @@
 #'
 #' @export
 
-estimate_weights <- function(data, centered_colnames = NULL, start_val = 0, method = "BFGS", ...) {
+estimate_weights <- function(data, centered_colnames = NULL, start_val = 0, method = "BFGS", nr.boot.iteration = NULL, set.seed.boot = 1234, ...) {
```

Comment:
```suggestion
estimate_weights <- function(data, centered_colnames = NULL, start_val = 0, method = "BFGS", n_boot_iterations = NULL, set_seed_boot = 1234, ...) {
```
maybe use underscores for consistency